db/migrations/create_chitter_table.sql

```diff
@@ -1,3 +1,4 @@
 CREATE DATABASE chitter;
 CREATE TABLE peeps(id SERIAL PRIMARY KEY, peeps VARCHAR(60));
--- ALTER TABLE peeps ADD COLUMN title VARCHAR(60);
\ No newline at end of file
+ALTER TABLE peeps ADD COLUMN timestamp timestamp with time zone;
+CREATE TABLE users (id SERIAL PRIMARY KEY, email VARCHAR(60), password VARCHAR(140));
```

Comment:
Might be missing something here but how did you manage to link users to their peeps without a foreign key?